### ISSUE TITLE
Swap to English / Chinese input mode by "SHIFT" key.

### ISSUE THREAD

**Issue by toniwong** [556556bt/HKCantonIME#8]
It is possible to disable swapping ENG/CHI by "SHIFT" key? It's troublesome I prefer switching it with "Control" + "Shift" between CHI & US keyboard. Thanks~!

**Comment by 556556bt** [comment]
yes, pressing "shift" key to swap between English and Chinese mode within the single IME follows consistency of other Chinese IME such as new changjei ime method. 
I would consider to have option to set HKCantonIME as purely Chinese input mode only.

**Comment by johnsonlam** [comment]
Well ... Google's input use SHIFT to temporary swap ENG/CHI mode is not ideal, since typing capital English letter will hit the SHIFT and change to Chinese, maybe use the Japanese key ALT+~ is much intuitive since almost no key conflict with others.

**Comment by johnsonlam** [comment]
I've maintain the Cantonese "Phrase Box" since 1994 until the ETen and Han Chinese system completely replaced by Windows, and I can't code, so nothing can be done. If I can put the phrase box in-use, I can keep on improving it.
